README.md

```diff
@@ -50,11 +50,14 @@ Postgrex is currently on 0.x beta releases. We are heading towards a stable rele
     uuid            <<160,238,188,153,156,11,78,248,187,109,107,185,189,56,10,17>>
     hstore          %{"foo" => "bar"}
     oid types       42
+    enum            'ok'
```

Comment:
It needs to be a string/binary, so it would be `"ok"`. `'ok'` is a list of characters, [?o, ?k].

Can you add the `***` after the ok so the comment below has a clearer reference?